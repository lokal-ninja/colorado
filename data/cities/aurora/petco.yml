id: 9041975619
type: node
latitude: 39.76331
longitude: -104.78022
postcode: "80011"
city: Aurora
street: "Salida Court"
housenumber: 3511
phone: ""
opening_hours: ""
website: ""
