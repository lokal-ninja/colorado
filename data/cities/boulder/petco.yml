id: 1270005948
type: node
latitude: 40.01348
longitude: -105.26186
postcode: "80302"
city: Boulder
street: "Arapahoe Avenue"
housenumber: 2480
phone: "+1 303-544-1888"
opening_hours: "Mo-Sa 09:00-20:00; Su 10:00-19:00"
website: "https://stores.petco.com/co/boulder/pet-supplies-boulder-co-1419.html"