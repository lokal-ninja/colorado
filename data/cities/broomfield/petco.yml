id: 369320538
type: way
latitude: 39.9172359
longitude: -105.0541558
postcode: "80020"
city: Broomfield
street: "Sheridan Boulevard"
housenumber: 12163
phone: ""
opening_hours: "Mo 09:00-19:00; Tu 10:00-20:00; We-Sa 09:00-21:00; Su 10:00-19:00"
website: ""
