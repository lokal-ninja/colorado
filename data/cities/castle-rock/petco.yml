id: 6917144671
type: node
latitude: 39.42239
longitude: -104.87635
postcode: "80108"
city: Castle Rock
street: "Promenande Parkway"
housenumber: 6378
phone: "+1 303-537-9000"
opening_hours: "Mo-Sa 09:00-20:00; Su 10:00-19:00"
website: "https://stores.petco.com/co/castlerock/pet-supplies-castlerock-co-2449.html"