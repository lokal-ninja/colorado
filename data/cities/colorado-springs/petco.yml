id: 4906461129
type: node
latitude: 38.84972
longitude: -104.84845
postcode: "80904"
city: Colorado Springs
street: "West Uintah Street"
housenumber: 1820a
phone: "+1-719-578-1123"
opening_hours: "Mo-Sa 09:00-21:00; Su 10:00-19:00"
website: "https://stores.petco.com/"
