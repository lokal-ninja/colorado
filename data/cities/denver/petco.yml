id: 273388802
type: node
latitude: 39.65179
longitude: -104.90293
postcode: "80237"
city: Denver
street: East Hampden Avenue
housenumber: "7400"
phone: +1 303-770-0607
opening_hours: Mo-Sa 09:00-20:00; Su 10:00-19:00
website: https://www.petco.com/
