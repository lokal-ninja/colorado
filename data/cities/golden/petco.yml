id: 92735977
type: way
latitude: 39.7222133
longitude: -105.19376190000001
postcode: "80401"
city: Golden
street: "West Colfax Avenue"
housenumber: 17132
phone: ""
opening_hours: ""
website: ""
